### DISCUSSION TITLE
Mismatch between m3u playlists and iptv-org website

### DISCUSSION THREAD

**Discussion by Poorchop** [iptv-org/community#76] [Q&A] [answered]
The website lists available streams grouped by country. Expanding the US category on the website will display channels that aren't available in the [us.m3u](https://iptv-org.github.io/iptv/countries/us.m3u) playlist.

For example, the Adult Swim Aqua Teen Hunger Force channel is categorized as a US stream on the website (TVG-ID `AdultSwimAquaTeenHungerForce.us`) but it does not exist in the us.m3u playlist. In fact, it seems to be grouped solely under the German playlist. The same is true of many other streams and I'm not quite sure what I'm doing wrong.

**Comment by BellezaEmporium** [comment]
It is normal.

The iptv-org website **is a database** of channels, and does not represents the channels we have in our playlists. Plain and simple.

**Reply by Poorchop** [reply]
Why is there a discrepancy in the metadata? For example, The Eric Andre Show is associated with the US and the English language in the database. There was even a point where [it was in the us.m3u playlist](https://github.com/iptv-org/iptv/commit/2d066924282a2896c222c039d6f524cb499c1c62). Now it is [in the German playlist](https://github.com/iptv-org/iptv/blob/d1ef7a5c34a485d244b782d4cfb8992bc25b7b13/streams/de_adultswim.m3u#L10) despite being an English language stream.

Is there any way that I can create a playlist that reflects how the database is organized? Thank you.

**Reply by BellezaEmporium** [reply]
There are two different versions of The Eric Andre Show Adult Swim stream, one in Germany, one in the US.

Depends on the one you're watching. If there's some dupes it should normally still be shown.

**Reply by Poorchop** [reply]
There aren't any dupes as far as I can tell. The Eric Andre Show only appears once in `index.m3u` and it has German metadata associated with it, although the show itself is presented in English. I still don't quite understand why the stream disappeared from the US playlists but I guess my only option at this point is to add the German playlist or `index.m3u` and hand pick other streams similarly.

**Comment by freearhey** [comment, marked as answer]
Fixed in #7641
